src/main/kotlin/blackjack/domain/Game.kt

```diff
@@ -0,0 +1,46 @@
+package blackjack.domain
+
+class Game(playerNames: String, whetherHit: (Player) -> Boolean) {
+    private val cardDeck: CardDeck = CardDeck()
+
+    private val _players: Set<Player>
+    val players: Set<Player>
+        get() = _players
+
+    init {
+        require(playerNames.isNotBlank()) { "playerNames is not blank." }
+
+        _players = playerNames.split(NAME_DELIMETER)
+            .map { Player(it, whetherHit) }
+            .toSet()
+    }
```

Comment:
굳이 변경할 필요는 없어보이지만 Backing Properties 대신 `by lazy` 키워드를 활용할수도 있겠네요.

```suggestion
    val players: Set<Player> by lazy {
        require(playerNames.isNotBlank()) { "playerNames is not blank." }

        playerNames.split(NAME_DELIMETER)
            .map { Player(it, whetherHit) }
            .toSet()
    }
```